dialogs.c

```diff
@@ -925,9 +927,22 @@ static glong date_compare_against_build_date(const char *iso8601_date)
 
 	build_time = g_date_time_to_unix(time);
 	ret = g_get_real_time() - build_time;
-
 	g_date_time_unref(time);
 
+#else
+
+	GTimeVal time;
+	gboolean parsed;
+
+	parsed = g_time_val_from_iso8601(iso8601_date, &time);
+	if (parsed) {
+		ret = time.tv_sec - build_time;
+	} else {
+		printf("%s could not parse date. Not a ISO 8601 format.", __func__);
```

Comment:
this one doesn't return anything here; as is above;
which is a inconsistent behavior between GLib versions;